### PR TITLE
create archetype files, _index cleanup

### PR DESCRIPTION
**Pull request by kbhawkey**
- Create archetype files for concepts, tasks
- Clean up tutorial _index page
- Add no_list parameter to contribute _index page
- Clean up setup _index page (Clarify that it is okay to remove the table in the index page)
- Removed example content and task pages
- See issue #22110.
- Create new pages, for example: `hugo new  docs/concepts/my-first-concept.md`
- Add info about creating new page

### CODE REVIEW

File: archetypes/tasks.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+content_type: task
+---
+
+<!-- overview -->
+
+## {{% heading "prerequisites" %}}
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
```
Comment (sftim) [inline archetypes/tasks.md:10]: `{{< version-check >}}` only really makes sense if `min-kubernetes-version:` is set in front matter.

Comment (kbhawkey) [reply archetypes/tasks.md:10]: OK. I took this directly from a task page. I think it is a "default" message warning readers to check their version.
I am undecided.

**Comment by kbhawkey** [comment]
Preview pages:
https://deploy-preview-22301--kubernetes-io-master-staging.netlify.app/docs/setup/
https://deploy-preview-22301--kubernetes-io-master-staging.netlify.app/docs/tasks/
https://deploy-preview-22301--kubernetes-io-master-staging.netlify.app/docs/tutorials/
https://deploy-preview-22301--kubernetes-io-master-staging.netlify.app/docs/contribute/
https://deploy-preview-22301--kubernetes-io-master-staging.netlify.app/docs/contribute/style/write-new-topic/#choosing-a-page-type

**Comment by tengqm** [comment]
/lgtm
Nice.

**Pull request merged**